integration/query_compat_test.go

```diff
@@ -239,44 +239,52 @@ func TestQueryCappedCollectionCompat(t *testing.T) {
 	require.Equal(t, compatInsertRes, targetInsertRes)
 
 	for name, tc := range map[string]struct {
-		filter bson.D
-		sort   bson.D
-		skip   string
+		filter          bson.D
+		sort            bson.D
+		skip            string
+		skipForFerretDB string
```

Comment:
I think we initially wanted to have `failsForFerretDB` everywhere. But personally I'm ok with `skip` too.